neovim/neovim
# tests: output_spec: fix "throttles shell-command output greater than ~10KB"

Latest failure for that test:
```
[ RUN      ] shell command :! throttles shell-command output greater than ~10KB: 
warning: Screen changes were received after the expected state. This indicates
indeterminism in the test. Try adding screen:expect(...) (or wait()) between
asynchronous (feed(), nvim_input()) and synchronous API calls.
  - Use screen:redraw_debug() to investigate; it may find relevant intermediate
    states that should be added to the test to make it more robust.
  - If the purpose of the test is to assert state after some user input sent
    with feed(), adding screen:expect() before the feed() will help to ensure
    the input is sent when Nvim is in a predictable state. This is preferable
    to wait(), for being closer to real user interaction.
  - wait() can trigger redraws and consequently generate more indeterminism.
    Try removing wait().
      
486.75 ms ERR
test/functional/ui/screen.lua:587: Failed to match any screen lines.
Expected (anywhere): "
%."
Actual:
  |1096: foo                                         |
  |1097: foo                                         |
  |1098: foo                                         |
  |1099: foo                                         |
  |[...]                                             |
  |                                                  |
  |{3:-- TERMINAL --}                                    |
stack traceback:
	test/functional/ui/screen.lua:587: in function '_wait'
	test/functional/ui/screen.lua:370: in function 'expect'
	test/functional/ui/output_spec.lua:61: in function <test/functional/ui/output_spec.lua:52>
```
(https://travis-ci.org/neovim/neovim/jobs/594313045#L6551)
